fuzz.go

```diff
@@ -0,0 +1,9 @@
+package jsonparser
+
+func FuzzParseString(data []byte) int {
+	r, err := ParseString(data)
+	if err != nil || r == nil {
```

Comment:
`/fuzz.go:5: invalid operation: r == nil (mismatched types string and nil)`

Should be `r == ""`